Apple-TV/PlaybackSettings.swift

```diff
@@ -0,0 +1,73 @@
+/*****************************************************************************
```

Comment:
I think this file shouldn't be inside Apple-TV folder.
Perhaps it should be inside SharedSources?

Reply:
true, But I think it is about time that we add swift to AppleTV